include/unused.hxx

```diff
@@ -44,10 +44,13 @@
 #if __has_cpp_attribute(maybe_unused)
 # define MAYBE_UNUSED(x) [[maybe_unused]] x
 #endif
-#elif defined(__GNUC__)
+#endif
+#ifndef MAYBE_UNUSED
+#if defined(__GNUC__)
 # define MAYBE_UNUSED(x) x __attribute__((unused))
```

Comment:
This could be `[[gnu::unused]] x` to make it look more like the maybe_unused branch above? This might also allow all the `MAYBE_UNUSED` calls to not take an argument (not sure if this is desirable).